src/__tests__/benchmark.test.js

```diff
@@ -178,3 +206,480 @@ test('Test select benchmark', async () => {
     'tpe_workers_1'
   );
 });
+
+test('Test (de)select assessments, tasks and algorithms', async () => {
```

Comment:
Do you think it would be better to separate each section in a different tests to each debugging it? Also I think it would be best to verify that the rest is still displayed when we deselect one item. For instance if the benchmark has AverageRanking and AverageResult, if we deselect AverageRanking, we test that AverageRanking is not displayed anymore, but we should also verify that AverageResult is still displayed.